src/home/apis/getPassword.ts

```diff
@@ -0,0 +1,34 @@
+import { authClient } from '@/apis';
+import { ErrorResponse, GetPasswordRespose, sessionToken } from '@/home/types/GetPassword.type';
+
+const authorization = (accessToken: string) => {
+  return { Authorization: `Bearer ${accessToken}` };
+};
+
+interface getPasswordProps {
+  password: string;
+  accessToken: string;
+}
+
+export const getPassword = async (props: getPasswordProps) => {
+  const { password, accessToken } = props;
+  if (!accessToken || !password) {
+    return null;
+  }
+  try {
+    const res = await authClient.get('/auth/verification/password', {
+      params: { password },
+      headers: authorization(accessToken),
+    });
+
+    if (res.status === 200) {
+      const data: GetPasswordRespose = { match: true, sessionToken: res.data as sessionToken };
+      console.log(data);
+      return data;
+    }
+  } catch (error: unknown) {
```

Comment:
`axios.get()` 함수는 제네릭 함수라 타입 매개변수를 넘기면 응답 데이터의 타입을 추론할 수 있습니다!

```suggestion
  try {
    const res = await authClient.get<sessionTokenType>('/auth/verification/password', {
      params: { password },
      headers: authorization(accessToken),
    });

    if (res.status === 200) {
      const data: GetPasswordRespose = { match: true, sessionToken: res.data };
      console.log(data);
      return data;
    }
  }
```

Reply:
타입스크립트 숙련도 이슈가.. ㅎㅎ 감사합니다!
반영했습니다. abd92ff9331fe4b1977a33dfb0e47025d060b902